manifests/init.pp

```diff
@@ -63,6 +63,9 @@
   $ssl_server_private   = undef,
   $ssl_client_certs     = 'puppet:///modules/mcollective/empty',
   $ssl_client_certs_dir = undef, # default dependent on $confdir
+
+  # Action policy settings
+  $allowunconfigured    = '1',
```

Comment:
would you mind changing this name to `allow_unconfigured` like the option is named as well?